js/mexc3.js

```diff
@@ -403,6 +405,11 @@ module.exports = class mexc3 extends Exchange {
                     'BEP20': 'BEP20(BSC)',
                     'BSC': 'BEP20(BSC)',
                 },
+                'networksById': {
+                    'BEP20(BSC)': 'BSC',
+                    'TRC20': 'TRX',
+                    'ERC20': 'ETH',
```

Comment:
@samgermain we should not add `ERC20:ETH` here. It will only be true when the currency is `ETH` and `networkCodeToId` already handles it for you. Going to remove it

Reply:
the same applies to `TRC20 and TRX`